### PR TITLE
CHAIN-164: add min fee for orders

### PR DESCRIPTION
**Pull request by bflood366**
If the quantity is too low then a `Quantity Too Low` error is shown on the Swap button. Same as how `Price Too Low` was handled. 

https://github.com/Chainring-Inc/chainring-monorepo/assets/162615268/f8cc29c0-4434-4df8-8061-b7fafb305f9d

### CODE REVIEW

File: sequencer/src/main/kotlin/co/chainring/sequencer/apps/SequencerApp.kt [hunk 1]
```diff
@@ -66,7 +66,15 @@ class SequencerApp(
...
                 if (state.markets.containsKey(marketId)) {
-                    error = SequencerError.MarketExists
+                    val currentMarket = state.markets.getValue(marketId)
+                    if (market.tickSize.toBigDecimal() != currentMarket.tickSize ||
+                        market.baseDecimals != currentMarket.baseDecimals ||
+                        market.quoteDecimals != currentMarket.quoteDecimals
+                    ) {
+                        error = SequencerError.MarketExists
+                    } else {
+                        currentMarket.minFee = market.minFee.toBigInteger()
```
Comment (bflood366) [inline sequencer/src/main/kotlin/co/chainring/sequencer/apps/SequencerApp.kt:76]: I made this change so we can call the sequencer AddMarkets again but have it just change the minFee as long as ticksize, base decimals and quote decimals are same as the created market.

Comment (antonyshchenko) [reply sequencer/src/main/kotlin/co/chainring/sequencer/apps/SequencerApp.kt:76]: This behaviour is a little counter-intuitive. Why not add a `SetMarketMinFee` call?

File: web-ui/src/components/Screens/HomeScreen/swap/SwapInternals.tsx [hunk 2]
```diff
@@ -70,6 +70,7 @@ export type SwapRender = {
   handleMaxBaseAmount: () => void
   handleMaxQuoteAmount: () => void
   topLimit: bigint | undefined
+  quantityTooLow: boolean
```
Comment (antonyshchenko) [inline web-ui/src/components/Screens/HomeScreen/swap/SwapInternals.tsx:73]: Wondering why you chose `quantity` term here and not `amount`?

**Review by antonyshchenko — APPROVED** [review]
Looks great!

File: sequencer/src/main/kotlin/co/chainring/sequencer/apps/SequencerApp.kt (hunk 1 above)
Comment (bflood366) [reply sequencer/src/main/kotlin/co/chainring/sequencer/apps/SequencerApp.kt]: added a `SetMarketMinFees` method

File: web-ui/src/components/Screens/HomeScreen/swap/SwapInternals.tsx (hunk 2 above)
Comment (bflood366) [reply web-ui/src/components/Screens/HomeScreen/swap/SwapInternals.tsx]: no reason - I changed to `amount` to be more consistent with other variable names

**Pull request merged**